cmd/soroban-rpc/internal/methods/get_events.go

```diff
@@ -159,7 +186,7 @@ func (e *EventFilter) Valid() error {
 	}
 	for i, topic := range e.Topics {
 		if err := topic.Valid(); err != nil {
-			return errors.Wrapf(err, "topic %d invalid", i+1)
+			return fmt.Errorf("topic %d invalid: %w", i+1, err)
```

Comment:
use [pkg/errors](https://github.com/pkg/errors) like I mentioned above to keep `errors.Is` compatibility and general Go-ness